Resources/views/change_password/index.html.twig

```diff
@@ -0,0 +1,39 @@
+{% block title %}{% endblock %}
+
+{% set hide_navigation_second = true %}
+{% set hide_page_header = true %}
+{% set page_class = 'login-container' %}
+
+{% block body %}
+    <div>
+
+        {% set formErrors = form.vars.errors.form.getErrors(true) %}
+
+        <div id="form_errors"
+             style="display: {% if formErrors is not empty %}block{% else %}none{% endif %};">
+            <h6 class="no-margin text-semibold">Er zijn een of meerdere fouten opgetreden:</h6>
+            <ul>
+                {% if formErrors is not empty %}
+                    {% for error in formErrors %}
+                        <li>{{ error.message }}</li>
+                    {% endfor %}
+                {% endif %}
+            </ul>
+        </div>
+
+        {% include '@WebstackUser/change_password/form.html.twig' %}
+    </div>
+    <div>
+        Het nieuwe wachtwoord dient te bestaan uit
```

Comment:
@rvmourik Even overleggen over welke eisen we aan een wachtwoord stellen in de front-end

Reply:
Ja hier gebruiken we een bundle voor toch? Geeft die ook niet iets van een "format" terug zodat we niet letterlijk html content hoeven typen voor de vereiste?

Reply:
De bundle die we gebruiken: https://github.com/rollerworks/PasswordStrengthBundle/ .Deze geeft echter geen human-readable password instructies zoals we die hier tonen.

Tevens zit er nog een ander issue aan die bundle (en Symfony zelf):
alle validators moeten op het plainPassword attribute;
https://github.com/rollerworks/PasswordStrengthBundle/issues/54#issuecomment-145822763
https://github.com/rollerworks/PasswordStrengthBundle/issues/28

Geldt ook voor de Symfony implementatie:
https://symfony.com/blog/new-in-symfony-4-3-compromised-password-validator

We hebben nu de password encoder geïmplementeerd zoals de maker bundle dat ook zou doen, maar dat werkt niet zonder die password encoder.